files/scripts/configdb-load.sh

```diff
@@ -10,4 +10,8 @@ if [ -r /etc/sonic/config_db.json ]; then
     sonic-cfggen -j /etc/sonic/config_db.json --write-to-db
 fi
 
+if [ -r /etc/sonic/minigraph.xml ]; then
+   sonic-cfggen -m -H --write-to-db
```

Comment:
>    [](start = 0, length = 3)

4 spaces

Reply:
we need to move this to vs start.sh before configdb-load.sh, the basic idea is to use minigraph to generate configdb, then here we can do configdb-load.sh

Reply:
Moved.